pkg/config/datafileprojectconfig/config.go

```diff
@@ -82,6 +82,24 @@ func (c DatafileProjectConfig) GetAttributeID(key string) string {
 	return c.attributeKeyToIDMap[key]
 }
 
+// GetAttributes returns attributes slice
+func (c DatafileProjectConfig) GetAttributes() []entities.Attribute {
+	rt := make([]entities.Attribute, 0)
```

Comment:
why putting 0, while we know the length.

Reply:
can you please give proper name instead of `rt`

Reply:
Is `ret` better?

Reply:
meaningful name can be attributes, return variable name should be functionally meaningful.